.travis.yml

```diff
@@ -27,11 +27,17 @@ before_script: # configure a headless display to test plot generation
     - "sh -e /etc/init.d/xvfb start"
     - sleep 3 # give xvfb some time to start
 
-script:
-    - flake8 parcels
-    - flake8 tests
-    - flake8 examples
-    - python parcels/scripts/pull_data.py
-    - py.test -v -s tests/
-    - py.test -v -s --nbval-lax examples/
-    - make linkcheck -C docs
+jobs:
+    include:
+        - script:
+            - flake8 parcels
+            - flake8 tests
+            - flake8 parcels/examples
```

Comment:
> However, then it doesn't really make sense to keep tests outside the parcels directory. Why not put that one in too?

I don't know. As `tests/` are not meant to be distributed together with Parcels, I'd leave them where they are. (I must admit, that I don't know whether there's any proper rationale for that or if I just think this is how it should be done because I've seen it elsewhere.)

> And isn't flake8 recursive? So is flake8 parcels/examples not redundant if we already do flake8 parcels?

That's right. I'll change it.

Reply:
@mlange05 and @delandmeterp; any thought on whether `tests` should then also go inside the `parcels` directory or not?

Reply:
I've noticed another problem: Currently, <https://github.com/OceanParcels/parcels/blob/467b25b110ee3bb34e4eb127586a7e7a24369bc1/tests/test_fieldset.py#L104> and <https://github.com/OceanParcels/parcels/blob/467b25b110ee3bb34e4eb127586a7e7a24369bc1/tests/test_scripts.py#L10> expect example data in `tests/../examples/`. This is not a problem, because I happened to call `parcels_get_examples examples` in Travis.  But in general, this is a problem.